domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/topic/domain/Topic.kt

```diff
@@ -0,0 +1,14 @@
+package com.krzykrucz.elesson.currentlesson.domain.topic.domain
+
+import com.krzykrucz.elesson.currentlesson.domain.NaturalNumber
+import com.krzykrucz.elesson.currentlesson.domain.NonEmptyText
+import com.krzykrucz.elesson.currentlesson.domain.attendance.CheckedAttendance
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.LessonIdentifier
+import java.time.LocalDate
+
+data class LessonOrdinalNumber(val number: NaturalNumber)
+data class FinishedLessonsCount(val count: NaturalNumber)
+data class TopicTitle(val title: NonEmptyText)
+data class LessonTopic(val lessonOrdinalNumber: LessonOrdinalNumber, val topicTitle: TopicTitle, val date: LocalDate)
```

Comment:
bearing in mind that we have some special representation for `NaturalNumber`, shouldn't `date` be express more specifically instead of just java `LocalDate`?

Reply:
Sounds interesting but I can't think of such a thing